### DISCUSSION TITLE
[BUG] QQ 老是掉线 而且 quick功能还不能正常用

### DISCUSSION THREAD

**Discussion by SmallXY** [NapNeko/NapCatQQ#661] [Show and tell]
### 系统版本

windows server

### QQNT 版本

无头的

### NapCat 版本

4.2.38

### OneBot 客户端

4.0+

### 发生了什么?

老是掉线  （大概1个小时左右就掉线）{"time":1735066588,"self_id":3389974256,"post_type":"notice","notice_type":"bot_offline","user_id":一个qq号,"tag":"下 线通知","message":"你的帐号当前登录已失效，请重新登录。"}
 quick功能还不能正常用

### 如何复现

我不知道

### 期望的结果?

别总是掉线 让他掉线能重连像早期mirai那种补救一下也行啊

### NapCat 运行日志

```shell
查不到不知道问什么
```

### OneBot 客户端运行日志

Active code page: 65001
Boot Command: "C:\NapCat\QQ.exe" --enable-logging -q 10086
[NapCat Backend] Main Process ID:5812
NapCat Shell App Loading...
12-25 03:51:51 [[32minfo[39m] [NapCat] [Core] NapCat.Core Version: 4.2.38
12-25 03:51:51 [[32minfo[39m] 核心登录服务连接成功!
12-25 03:51:51 [[31merror[39m] 快速登录失败，未找到该 QQ 历史登录记录，将使用二维码登录方式
12-25 03:51:51 [[32minfo[39m] [NapCat] [WebUi] WebUi Local Panel Url: http://127.0.0.1:6099/webui?token=e7ing658538
12-25 03:51:51 [[32minfo[39m] [NapCat] [WebUi] WebUi Local Panel Url: https://napcat.152710.xyz/web_login?back=http://127.0.0.1:6099&token=e7ing658538
12-25 03:51:53 [[33mwarn[39m] 请扫描下面的二维码，然后在手Q上授权登录：
12-25 03:51:53 [[33mwarn[39m]

▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
█ ▄▄▄▄▄ █▄▄▄ ▀▄██ ▄  ▀▄▀█ █ ▄▄▄▄▄ █
█ █   █ ██▄▀ █ ▀▄▄▀█ ▀    █ █   █ █
█ █▄▄▄█ ██▀▄ ▄████  ███ ▀ █ █▄▄▄█ █
█▄▄▄▄▄▄▄█ ▀▄█ ▀ █ █▄▀▄█ ▀ █▄▄▄▄▄▄▄█
█▄▄█  █▄▀█▄▀█▄██▀▀▄▄▀█▀▀█▄█▄▀▀█▀▀▄█
█▀▀▀▀▀ ▄▀▀▄██▄█▄ █  █ ▄▄▄▀▀  ▄▀▀▄▄█
██▀ ▀█ ▄██  █▀▀▄  ▄▀▄▀█ ▀▄▀▄▄███▄▄█
█▀▀▀▀ ▀▄█▀▄▀█▀▄▀ ▄█▀ ▄█▄██▀▄█ █▀█▀█
█▀█▀ ▄▄▄ ▄ █ ▄▄  ▀▀▄▀▀█▄▄█▄ ▀ ▄█  █
█▀▄▀ ▄█▄ █▀█▀▄    █ ▄ ▄▄▄██▀█▄█▀▀ █
█ ███ █▄▄▄██▄▀██▄█▀▀█▄▄█ ▀▄█▀▀▀█▀██
█ █▀▄█▄▄▄█▀▀ ▀█ ▄▄ ████ ▄▀ ▀ ▄█▀▄▄█
█▄▄▄█▄█▄█▀█ ▄▄▀▄▄ █▀▀█▀▀█ ▄▄▄ ▀▀ ██
█ ▄▄▄▄▄ ██   ▄█  ▄█▀  ▄   █▄█ ▀ █▀█
█ █   █ █ █▄▀ ▄ ▀█ ▀▀ ▄▄▄▄ ▄▄▄▀█▄▄█
█ █▄▄▄█ █▀▄█▄▄    ▄▀▄▀▄ ▀█▀▀▄   ▀ █
█▄▄▄▄▄▄▄█▄▄█████▄███▄▄█▄█▄▄█▄█▄████

二维码解码URL: https://txz.qq.com/p?k=ukB*49m3pVxQ3cYZw4lHAa0D2MdDVnEV&f=1600001604
如果控制台二维码无法扫码，可以复制解码url到二维码生成网站生成二维码再扫码，也可以打开下方的二维码路径图片进行扫码。
12-25 03:51:53 [[33mwarn[39m] 二维码已保存到 C:\NapCat\versions\9.9.16-29927\resources\app\napcat\cache\qrcode.png
12-25 03:52:09 [[32minfo[39m] 正在快速登录  3389974256
12-25 03:52:15 [[34mdebug[39m] 本账号数据/缓存目录： C:\Users\Administrator\Documents\Tencent Files\NapCat\data
12-25 03:52:15 [[34mdebug[39m] [Core] [Config] 配置文件C:\NapCat\versions\9.9.16-29927\resources\app\napcat\config\napcat_3389974256.json加载 {"fileLog":false,"consoleLog":true,"fileLogLevel":"debug","consoleLogLevel":"info","packetBackend":"auto","packetServer":""}
12-25 03:52:15 [[33mwarn[39m] [Core] [Packet] wsPacketClient 未配置服务器地址
12-25 03:52:15 [[32minfo[39m] [Core] [Packet] 自动选择 NativePacketClient 作为后端
12-25 03:52:15 [[32minfo[39m] [Notice] [OneBot11] [network] 配置加载
HTTP服务: 0.0.0.0:3000, : 已启动
WebSocket服务: 0.0.0.0:3001, : 已启动

12-25 03:52:15 [[32minfo[39m] [OneBot] [WebSocket Server] Server Started :::3001
12-25 03:52:15 [[32minfo[39m] [OneBot] [HTTP Server Adapter] Start On Port 3000
201 js loaded
201 js loaded

**Comment by theed4k** [comment]
我也是老是掉线，但是不至于1个小时就掉线，大概6-10个小时左右

**Comment by AileYoung233** [comment]
我也掉线，看来这几天大家都有问题啊

**Comment by SaarChaffee** [comment]
这几天开始会弹 社交功能限制提醒

**Comment by miiiiiint** [comment]
腾讯开始发力了

**Comment by ERYhua** [comment]
同掉线问题，且快速登陆使用不了

**Comment by ELDment** [comment]
年末了 腾讯发力了🤓👌

**Comment by chenzhimin4213** [comment]
寻找一位能逆向编辑的作者过TP检测系统

**Comment by pk5ls20** [comment]
尝试更新最新版本

**Comment by youli42** [comment]
跟个帖，最新版登录，过了几天再登录被风控了，
验证后再登录，每天晚上就被下线，挺难受的

**Comment by MoonShadow1976** [comment]
+1, 账号风控
